src/Orleans.Transactions.TestkitBase/TestRunners/TransactionalStateStorageTestRunner.cs

```diff
@@ -133,7 +133,7 @@ public virtual async Task WrongEtags()
             loadresponse.PendingStates.Should().BeEmpty();
         }
 
-        private PendingTransactionState<TState> MakePendingState(long seqno, int val, bool tm)
+        private PendingTransactionState<TState> MakePendingState(long seqno, TState val, bool tm)
```

Comment:
>TState val [](start = 77, length = 10)

You can make this value the seed parameter and just call the state factory within MakePendingState()